odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1278,7 +1284,7 @@ def distribute_branding(self, e, branding=None, parent_xpath='',
                 # running index by tag type, for XPath query generation
                 indexes = collections.defaultdict(lambda: 0)
                 for child in e.iterchildren(tag=etree.Element):
-                    if child.get('data-oe-xpath'):
+                    if child.get('data-oe-xpath') or child.get('data-oe-field-xpath'):
```

Comment:
Why use a new and separate attribute, is using `data-oe-xpath` for fields as well not ok?

Reply:
As I am fixing in 12.0, I was being cautious but yes using data-oe-xpath would require more changes. For example, just for saving an edited field, knowing it is a field or a view is known thanks to the non presence of data-oe-xpath. I can adapt of course, but what about all potential customization relying on this?

Reply:
That seems like a fair enough reason to use a separate attribute, but it should be documented in the commit message for the enlightenment of future generations (and whoever wonders about this 2 weeks from now)

Reply:
Ok, I will document that, then merge after a bit more testing and once the forward-port reaches master, I'll try using a common attribute :+1: